redash/query_runner/salesforce.py

```diff
@@ -78,7 +78,7 @@ def configuration_schema(cls):
                     "default": DEFAULT_API_VERSION,
                 },
             },
-            "required": ["username", "password", "token"],
+            "required": ["username", "password"],
             "secret": ["password", "token"],
```

Comment:
If you remove `token` from required, you should remove it from secret too.
```suggestion
            "secret": ["password"],
```